src/website/src/pages/eu/components/table/docs/code.mdx

```diff
@@ -34,3 +40,21 @@ import { tableDefault, tableMulti, tableSortable, tableZebra } from '../demo';
 >
   <Html markup={tableSortable} />
 </Playground>
+
+## Enhanced mobile table
+
+Enhanced tables offers a more user friendly display on mobile.
```

Comment:
But why we don't specify what are the "tables" where this can be safely enabled..? I mean, for this to work we need the first tr to contain th(s) as a basic requirement.
Then this might not be enough in 100% of the cases, so this is why making this optional.